shared/devices/render.desktop.js

```diff
@@ -108,11 +109,16 @@ class Render extends Component<void, Props, State> {
   }
 
   _items () {
-    return [
-      {title: 'New Phone', onClick: () => this.props.addNewPhone()},
+    const items = [
       {title: 'New Computer', onClick: () => this.props.addNewComputer()},
       {title: 'New Paper Key', onClick: () => this.props.addNewPaperKey()},
     ]
```

Comment:
@chrisnojima that's what I'd tried first, doesn't flow type :)
```
devices/render.desktop.js:113
113:       ...(flags.mobileAppsExist && [{title: 'New Phone', onClick: () => thi
s.props.addNewPhone()}]),
               ^^^^^^^^^^^^^^^^^^^^^ boolean. This type is incompatible with
113:       ...(flags.mobileAppsExist && [{title: 'New Phone', onClick: () => thi
s.props.addNewPhone()}]),
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
^^^^^^^^^^^^^^^^^^^^^^^ spread operand
```
It looks like @chromakode's doesn't flow type either, though I'd guessed it would:
```
devices/render.desktop.js:124
124:         <DeviceHeader
             ^ React element `DeviceHeader`
113:       flags.mobileAppsExist ? {title: 'New Phone', onClick: () => this.prop
s.addNewPhone()} : null,
                                                                                
                   ^^^^ null. This type is incompatible with
 15:   items: Array<MenuItem | 'Divider'>, // Divider is not used on native
                    ^^^^^^^^^^^^^^^^^^^^ union: MenuItem | string literal `Divid
er`. See: common-adapters/popup-menu.js.flow:15
```